action.yml

```
@@ -23,6 +23,9 @@ inputs:
   body:
     description: Body of mail message (might be a filename prefixed with file:// to read from)
     required: true
+  html_body:
+    description: HTML part body of multipart mail message (might be a filename prefixed with file:// to read from)
+    required: true
```

```suggestion
    required: false
```

It shouldn't be required right?